common/chunkStatusLogger.go

```diff
@@ -241,6 +241,7 @@ type ChunkStatusLoggerCloser interface {
 	GetCounts(td TransferDirection) []chunkStatusCount
 	GetPrimaryPerfConstraint(td TransferDirection, rc RetryCounter) PerfConstraint
 	FlushLog() // not close, because we had issues with writes coming in after this // TODO: see if that issue still exists
+	CloseLogger()
```

Comment:
We cannot disregard the TODO warning in FlushLog() as it explicitly calls out why they don't have a Close.
Either we mention here that we do a deferred close to workaround this problem or something else.